packages/contracts-governance/contracts/MintManager.sol

```diff
@@ -15,7 +15,7 @@ contract MintManager is Ownable {
     /**
      * @notice The GovernanceToken that the MintManager can mint tokens
      */
-    GovernanceToken public governanceToken;
+    GovernanceToken public immutable governanceToken;
```

Comment:
I see the `mintPermittedAfter` is declared after this, so we probably need a spacer here?

Reply:
No we don't - its not deployed behind a proxy. We need to call `upgrade()` on the existing mint manager with the address the new mint manager is deployed to

Reply:
Ah, gotcha.